docs/UserGuide.md

```diff
@@ -76,6 +76,7 @@ _With MediConnect, managing your healthcare practice has never been easier._
 | Field            | Prefix | Input Restrictions                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 |------------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **Address**      | a/     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| **Cost**         | c/     | Should either be a positive number or 0. Supports up to 2 decimal places.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Could perhaps phrase this as "1. Should be a number larger or equal to 0." "2. Supports up to 2 decimal places."
to be more consistent with the rest of the table.